rosdep/python.yaml

```diff
@@ -1966,10 +1966,19 @@ python-grpcio:
     '*': [python-grpcio]
     bionic:
       pip: [grpcio]
+    focal:
+      pip: [grpcio]
```

Comment:
Since the default ROS installation on Ubuntu Focal targets python3. I'm wondering if we should create a new key, `python3-grpcio-pip`. It also appears that this package is now available directly from the Ubuntu repositories for Focal https://packages.ubuntu.com/focal/python3-grpcio. Would it make sense to switch away from the pip key entirely?

Reply:
I just realized there is already an entry for `python3-grpcio` https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml#L5574. I can add `focal` there if it is better.

Reply:
> I just realized there is already an entry for `python3-grpcio` https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml#L5574. I can add `focal` there if it is better.

That shouldn't be necessary; the wildcard syntax there already covers focal.  So in point of fact, this part of the change shouldn't be necessary at all.